src/harberger-lib/components/Info.re

```diff
@@ -240,3 +241,168 @@ let make = (~tokenId: TokenId.t) => {
      }}
   </React.Fragment>;
 };
+
+module Auction = {
+  [@react.component]
+  let make = (~tokenId: TokenId.t, ~abandoned: bool, ~auctionStartTime) => {
+    let currentPatron = QlHooks.usePatron(tokenId) |||| "Loading";
+    let displayName = UserProvider.useDisplayName(currentPatron);
+    let displayNameStr = UserProvider.displayNameToString(displayName);
+
+    let tokenName = tokenId->QlHooks.useWildcardName |||| "loading name";
+
+    let clearAndPush = RootProvider.useClearNonUrlStateAndPushRoute();
+
+    let currentUsdEthPrice = UsdPriceProvider.useUsdPrice();
+    let (totalPatronage, totalPatronageUsd) =
+      QlHooks.useAmountRaisedToken(tokenId)
+      ->mapd(("Loading", "Loading"), a =>
+          (
+            (a->Eth.get(Eth.Eth(`ether))->Float.fromString |||| 0.0)
+            ->toFixedWithPrecisionNoTrailingZeros(~digits=9),
+            currentUsdEthPrice->mapd("Loading", usdEthRate =>
+              a->Eth.get(Eth.Usd(usdEthRate, 2))
+            ),
+          )
+        );
+
+    let ratio = QlHooks.usePledgeRate(tokenId);
+
+    let monthlyRate = Js.Float.toString(ratio *. 100.);
+
+    <React.Fragment>
+      <div>
+        {if (ratio == 0.) {
+           <p>
+             "The monthly pledge rate will be revealed at launch."->restr
+           </p>;
+         } else {
+           <>
+             <small>
+               <strong>
+                 "Monthly Pledge Rate:"->restr
+                 <Rimble.Tooltip
+                   message={
+                     "This is the monthly percentage contribution of "
+                     ++ tokenName
+                     ++ "'s sale price that will go towards conservation of at risk animals. This is deducted continuously from the deposit and paid by the owner of the animal"
+                   }
+                   placement="top">
+                   <span> {js|ⓘ|js}->restr </span>
+                 </Rimble.Tooltip>
+               </strong>
+             </small>
+             <br />
+             {(monthlyRate ++ " %")->restr}
+           </>;
+         }}
+      </div>
+      {abandoned
+         ? <p>
+             <strong>
+               "Was abandoned by "->restr
```

Comment:
I actually think this is a bit harsh, maybe change this to 'previous owner' and have the name show below as text not in the heading.